java/pom.xml

```diff
@@ -35,83 +35,313 @@
     </license>
   </licenses>
   <properties>
-    <java.version>1.8</java.version>
+    <!-- https://cloud.google.com/dataproc-serverless/docs/concepts/versions/spark-runtime-versions#spark_runtime_version_12 -->
+    <!-- Java & Scala Version Aligned with Dataproc Serverless Spark Runtime 1.2 -->
+    <java.version>17</java.version>
```

Comment:
First we need to submit the upgrade to version java 11 + dataproc serverless 1.2

Reply:
I am in support of JDK 17 instead of JDK 11. I have seen lots of people have been already migrated to JDK 17. I noticed lots of questions on stackoverflow for Dataproc 2.x

Even I am in support of starting to think about dataproc 2.x migration

Reply:
We can't go to java 17 directly, since we got to support both S8S v1.1 and S8S v1.2 for some time (until the UI and downstream is moved to 1.2).
Therefore we need to change to Java 11

Reply:
I am trying to downgrade it with Java 11.